README.md

```diff
@@ -77,7 +77,7 @@ devtools::load_all()
 aNCA::run_app()
 ```
 
-Have fun running your tailored Non-Compartemental Analysis!
+Have fun running your tailored Non-Compartmental Analysis!
```

Comment:
question: I can see that there are still a few places in the repo that use `Non-Compartment Analysis`. Should they be changes to `Non-Compartmental Analysis` as well in the scope of this PR?